sources/packages/api/src/database/entities/student-appeal-requests.model.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { Note, StudentAppeal, StudentAppealStatus, User } from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+
+/**
+ * Represents as individual appeal requested by a student, for instance,
+ * to have his income or dependents data changed on his Student Application
+ * after it was completed.
+ */
+@Entity({ name: TableNames.StudentAppealRequest })
```

Comment:
`StudentAppealRequest**s**`?

Reply:
We do have both but I agree to keep the constants more aligned with the real table name.
![image](https://user-images.githubusercontent.com/61259237/157290052-be2669f4-ca2a-48af-b7f2-010e6b5a1519.png)